recipes.js

```diff
@@ -1,10 +1,100 @@
-const mongoose = require('mongoose');
-const Schema   = mongoose.Schema;
-const data = require('./data.js');
+// Setup
+// -------------------------------------------------------------------------------------------
 
-mongoose.connect('mongodb://localhost/recipeApp')
+const mongoose = require("mongoose");
+const Schema = mongoose.Schema;
+const data = require("./data.js");
+
+mongoose
+  .connect("mongodb://localhost/recipeApp")
   .then(() => {
-    console.log('Connected to Mongo!');
-  }).catch(err => {
-    console.error('Error connecting to mongo', err);
+    console.log("Connected to Mongo!");
+  })
+  .catch(err => {
+    console.error("Error connecting to mongo", err);
+  });
+
+// use the Schema class to create our recipe schema object
+// (the schema is the STRUCTURE of documents in the model's collection)
+const recipesSchema = new Schema({
+  title: { type: String, required: true, unique: true },
+  level: {
+    type: String,
+    enum: ["Easy Peasy", "Amateur Chef", "UltraPro Chef"]
+  },
+  ingredients: [],
+  cuisine: { type: String, required: true },
+  dishType: {
+    type: String,
+    enum: ["Breakfast", "Dish", "Snack", "Drink", "Dessert", "Other"]
+  },
+  image: {
+    type: String,
+    default: "https://images.media-allrecipes.com/images/75131.jpg"
+  },
+  duration: { type: Number, min: 0 },
+  creator: { type: String },
+  created: { type: Date, default: Date.now }
+});
+
+// the variable "Recipe" is our Mongoose model class
+// the "Recipe" model will allow is to make queries on the "recipes" collection
+// (Mongoose turns the model's name string "Dog" to the collection names "recipes")
+const Recipe = mongoose.model("Recipes", recipesSchema);
+
+// share the "Recipe" variable with other files that require recipes.js
+// (Recipe is our Mongoose model that connects to the "recipes" collection)
+// module.exports = Recipe;
+
+// Routes
+// -------------------------------------------------------------------------------------------
+
+// (C)reate
+Recipe.create({ title: "Asian Glazed Chicken Thighs", cuisine: "Asian" })
```

Comment:
You have an small error here. In your model, you say that the names need to be unique but you're using the same name as one of the items in the data. 

you get this error message "E11000 duplicate key error collection: recipeApp.recipes index: title_1 dup key: { : "Asian Glazed Chicken Thighs" }"